app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(user_id: @user.id, post: @post, text: params[:comment][:text])
+    @comment.save
+    redirect_to user_post_path(@user, @post)
```

Comment:
Great job so far @asifaminisonline 
**OPTIONAL**

- [ ] Please create a [Private Params Method](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md) to secure your user form inputs.